scripts/generate/constants.ts

```diff
@@ -1,3 +1,6 @@
 // corresponds to Extension::Model::Type.identifier
-// https://github.com/Shopify/shopify-app-cli-extensions/tree/master/lib/project_types/extension/models/types
-export const extensionTypes = ['SUBSCRIPTION_MANAGEMENT'];
+// https://github.com/Shopify/shopify-app-cli/tree/master/lib/project_types/extension/models/types
+export const extensionTypes = [
+  'PRODUCT_SUBSCRIPTION',
+  'CHECKOUT_POST_PURCHASE',
```

Comment:
Why do we need `CHECKOUT_POST_PURCHASE` here? It should belong to argo-checkout-template.